File: Android/app/src/main/java/com/arun/api/Model/RequestForm.java
package com.arun.api.Model;

public class RequestForm {
     int Id;
     String Comments;
     int FormStatus;

    public int getHandledBy() {
        return HandledBy;
    }

    public void setHandledBy(int handledBy) {
        HandledBy = handledBy;
    }

    int HandledBy;

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public String getComments() {
        return Comments;
    }

    public void setComments(String comments) {
        Comments = comments;
    }

    public int getFormStatus() {
        return FormStatus;
    }

    public void setFormStatus(int formStatus) {
        FormStatus = formStatus;
    }
}

File: SSIS/SSIS/Models/Util Models/RequestForm.cs
using SSIS.Enums;

namespace SSIS.Models
{
    public class RequestForm
    {
        public int Id { get; set; }
        public string Comments { get; set; }
        public FormStatus FormStatus { get; set; }
        public int HandledBy { get; set; }
    }
}
